main.js

```diff
@@ -0,0 +1,26 @@
+    let els = document.querySelectorAll("header nav ul li");
+    for(let el of els){
+        if(el.textContent.includes("All Posts")){
+            el.remove();
+        }
+    }
+
+    let spans = document.querySelectorAll(".featured span");
+    for(let span of spans){
+        if(span.textContent.includes("Read Time")){
+            span.remove();
+        }
+    }
```

Comment:
```suggestion
document.querySelector("span").remove();
```
This one can also be done in one line, because all of the `<span>` elements happen to only be used for "Read Time". Again, if that were not the case, and `<span>` tags were being used in other places, the for loop **_would_** be necessary.